src/main/java/com/microsoft/sqlserver/jdbc/SQLServerXAResource.java

```diff
@@ -479,7 +479,7 @@ private XAReturnValue DTC_XA_Interface(int nType, Xid xid, int xaFlags) throws X
                              */
                             String buildInfo = rs.getString(4);
                             // SQL Server Linux is x64-compatible only.
-                            if (null != buildInfo && buildInfo.contains("Linux")) {
+                            if (null != buildInfo && (buildInfo.contains("Linux") || buildInfo.contains("Microsoft SQL Azure"))) {
```

Comment:
Does this imply that Azure SQL is always going to be 64-bit architecture? Do you have documentation that shows this?